mvpa2/generators/resampling.py

```diff
@@ -203,3 +203,71 @@ def generate(self, ds):
     def __str__(self):
         return _str(self, str(self._amount), n=self._attr, count=self.count,
                     apply_selection=self._apply_selection)
+
+
+class NonContiguous(Node):
+    """Generator to remove samples too close as measured with a sample attribute
+    eg.:
+    >>> partitioner = ChainNode([NFoldPartitioner(),
+    NonContiguous(attr='time',
+    dist=60,
+    partition_keep=2,
+    partition_trim=1)])
+    if sa.time in sec then all the training samples in partition(_keep)=1
+    that are less than 60 secs distant from partition(_trim)=2 
+    will be assigned to partition 0 to be excluded from cross-validation fold
+    with appropriate splitter
+    """
+    def __init__(self,
+                 dist_attr='chunks',
+                 dist=1,
+                 split_attr='partitions',
+                 split_keep=[2],
+                 split_trim=[1],
+                 split_assign=[0],
+                 **kwargs):
+        """
+        Parameters
+        ----------
+        dist_attr : the attribute to measures distance
+        dist : the minimum distance between the samples to the 2 splits of data
+        split_attr : the attribute describing the split
+        split_keep : the list of splits that are to be kept : default [2] (testing)
+        split_trim : the list of splits to be trimmed : default [1] (training)
+        split_assign : the value to be assigned to trim samples
```

Comment:
same here -- ideally should follow the NumPy convention for parameters description here.  Just see other docstrings in the code